sparse/etc/dconf/db/vendor.d/lipstick-configs-kirin.txt

```diff
@@ -0,0 +1,6 @@
+[desktop/lipstick-jolla-home]
+reboot_warning_on_sim_remove=true
+
+[desktop/lipstick-jolla-home/peekfilter]
```

Comment:
Not sure if this is still needed, maybe @pvuorela or @rainemak knows?

Reply:
Looks like still used.